src/objective/objective.cc

```diff
@@ -31,6 +31,11 @@ ObjFunction* ObjFunction::Create(const std::string& name, GenericParameter const
   return pobj;
 }
 
+void ObjFunction::InitEstimation(MetaInfo const&, linalg::Tensor<float, 1>* base_score) const {
```

Comment:
So, at this time, default behaviour will be to use the initialisation for absolute error but not other loss functions? Should we add this for all loss functions? I think it makes sense, but we might need to manage user expectations when their model accuracy changes substantially between versions.

Reply:
I plan to work on other functions as well.  This PR is just to get started with basic stuff like model serialization.  The biggest issue for now is multi class model where we need a vector of base scores.

Reply:
I have a prototype for least square. Will follow up once this PR is sorted out.